### PR TITLE
Install OpenMPI for CUDA 10 when not present in the system

### PR DESCRIPTION
**Pull request by jantonguirao**
Signed-off-by: Joaquin Anton <janton@nvidia.com>

### CODE REVIEW

**Comment by jantonguirao** [comment]
!build

File: qa/L1_tensorflow-dali_test/test.sh [hunk 1]
```diff
@@ -11,12 +11,15 @@ NUM_GPUS=$(nvidia-smi -L | wc -l)
...
-# MPI is present in CUDA 10 image already so no need to build it
 if [ "${CUDA_VERSION}" == "100" ]; then
     pip install tensorflow-gpu==1.13.1
 else
     pip install tensorflow-gpu==1.12
+fi
 
+# MPI might be present in CUDA 10 image already so no need to build it if that is the case
+if [ ! -d "/usr/local/mpi" ]; then
+    apt-get update && apt-get install -y wget
```
Comment (szalpal) [inline qa/L1_tensorflow-dali_test/test.sh]: What if MPI is present, but not in `/usr/local//mpi`? Is it possible?
Or `/usr/local/mpi` turns out to be empty? ;)

Comment (jantonguirao) [reply qa/L1_tensorflow-dali_test/test.sh]: Then the test will misbehave. 
Do you have a suggestion for a more appropriate check?

Comment (JanuszL) [reply qa/L1_tensorflow-dali_test/test.sh]: Maybe just `which ompi_info`?

Comment (jantonguirao) [reply qa/L1_tensorflow-dali_test/test.sh]:
```
if ! [ -x "$(command -v mpicxx)" ]; then
```
works

**Comment by jantonguirao** [comment]
!build

**Pull request merged**